engine/src/main/scala/cromwell/engine/workflow/ShadowWorkflowActor.scala

```diff
@@ -76,7 +77,7 @@ object ShadowWorkflowActor {
     */
   case object WorkflowFailedState extends ShadowWorkflowActorTerminalState
 
-  case class ShadowWorkflowActorData(backendAssignments: Map[Call, String])
+  case class ShadowWorkflowActorData(workflowDescriptor: Option[EngineWorkflowDescriptor])
```

Comment:
Could we have a case object for the empty case and the class for the descriptor-ful case?